src/main/kotlin/no/nav/syfo/dialogmote/database/repository/MoteStatusEndretRepository.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.syfo.dialogmote.database.repository
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.application.database.toList
+import no.nav.syfo.dialogmote.database.domain.PMoteStatusEndret
+import no.nav.syfo.dialogmote.database.domain.toDialogmoteStatusEndret
+import no.nav.syfo.dialogmote.domain.DialogmoteStatus
+import no.nav.syfo.dialogmote.domain.DialogmoteStatusEndret
+import no.nav.syfo.domain.PersonIdent
+import java.sql.ResultSet
+import java.time.LocalDateTime
+import java.util.*
+
+class MoteStatusEndretRepository(private val database: DatabaseInterface) {
+
+    fun getMoteStatusEndringer(personident: PersonIdent): List<Triple<DialogmoteStatusEndret, String, Pair<LocalDateTime, String>>> =
```

Comment:
Det peneste hadde kanskje vært å lage en egen type, men jeg slet litt med å finne hva slags domene-konsept det i så fall skulle vært, siden vi sammenfatter litt forskjellig data her. Evt bare krype til korset og kalle det `MoteStatusEndretHistorikk`, selv om jeg ikke føler det er et domene på samme måte.

Reply:
Synes vi godt kunne definert et separat historikk domene-konsept-type om enn bare for å slippe sånn hårete Triple-struktur :cry:

Reply:
Blir ikke alt litt enklere om modellen her flates ut - så lenge man har med en moteUuid sånn at man vet hvilke statusendringer som gjelder samme møte så burde det være ok. Hvis man vil ha med hvem som opprettet møtet så er det vel ikke så stort problem om det er med i alle radene som har med det møtet å gjøre?

Reply:
Kom med en liten endring nå, der jeg tok inn DTOen direkte her allerede. Det kræsjer litt med clean architecture tankegangen 🤔